enterprise/app/services/enterprise/message_templates/hook_execution_service.rb

```diff
@@ -2,6 +2,7 @@ module Enterprise::MessageTemplates::HookExecutionService
   def trigger_templates
     super
     return unless should_process_captain_response?
+    return perform_handoff unless inbox.captain_active?
```

Comment:
we will need to do it inside captain as there could be customers who have connected dialogflow or agents bots

Reply:
never mind, the should_process_captain_response? will cover this.